examples/gno.land/r/gov/dao/v3/init/init.gno

```diff
@@ -0,0 +1,27 @@
+package init
+
+import (
+	"std"
+
+	"gno.land/r/gov/dao"
+	"gno.land/r/gov/dao/v3/impl"
+	"gno.land/r/gov/dao/v3/memberstore"
+)
+
+func Init() {
+	dao.UpdateImpl(dao.UpdateRequest{
+		DAO:         impl.NewGovDAO(),
```

Comment:
```suggestion
		DAO:         impl.DAO,
```

What about expecting each implementation to expose a global variable and using this Init() function to handle the mapping instead of creating it here?